### DISCUSSION TITLE
[FEATURE REQ] Add Signal as a Notification Type

### DISCUSSION THREAD

**Discussion by yoyo292949158** [autobrr/autobrr#1752] [Ideas]
It would be great if there was an option to select Signal for use as the notification type.

I am not sure how portable/reusable the existing code is, but this functionality seems to exist within Radarr and Sonarr as a notification agent. It does require the user to also run [Signal-CLI](https://github.com/AsamK/signal-cli)

Thanks for your consideration.

**Comment by NoX1De** [comment]
See useful info in https://github.com/autobrr/autobrr/discussions/2296, +1 on this please add the ability to use Signal for notifications!
